examples/gno.land/r/demo/boards2/render.gno

```diff
@@ -59,7 +59,9 @@ func renderThread(res *mux.ResponseWriter, req *mux.Request) {
 	board := v.(*Board)
 	thread, found := board.GetThread(PostID(tID))
 	if !found {
-		res.Write("Thread does not exist with ID: " + req.GetVar("thread"))
+		res.Write("Thread does not exist with ID: " + rawID)
+	} else if thread.Hidden() {
+		res.Write("Thread with ID: " + rawID + " was hidden")
```

Comment:
Maybe we could use something like for example "has been flagged as inappropriate" when mentioning that a thread or reply is hidden:

```suggestion
		res.Write("Thread with ID " + rawID + " has been flagged as inappropriate")
```